src/osd/OSD.cc

```diff
@@ -2047,13 +2049,19 @@ OSD::OSD(CephContext *cct_, ObjectStore *store_,
 OSD::~OSD()
 {
   delete authorize_handler_cluster_registry;
+  authorize_handler_cluster_registry = nullptr;
```

Comment:
what is the point of setting a member variable to null here? as i think it is a dtor?

Reply:
@shinobu-x this is not addressed.